traffic_ops/testing/api/v14/config/cache_config/cache_config.go

```diff
@@ -0,0 +1,265 @@
+/*
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package cache_config
+
+import (
+	"regexp"
+	"strings"
+
+	. "github.com/apache/trafficcontrol/traffic_ops/testing/api/v14/config"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/test"
+	"github.com/go-ozzo/ozzo-validation/is"
+)
+
+// ParseCacheConfig takes a string presumed to be an ATS cache.config and validates
+// it is syntatically correct.
+//
+// The general format of a cache config is three types of labels separated by spaces:
+//
+//  primary_destination=value secondary_specifier=value action=value
+//
+// For a full description of how to format a cache config, refer to the ATS documentation
+// for the cache config:
+// https://docs.trafficserver.apache.org/en/latest/admin-guide/files/cache.config.en.html
+//
+func ParseCacheConfig(config string) test.Error {
+	lines := strings.Split(config, "\n")
+
+	if len(lines) == 1 {
```

Comment:
This one-off isn't necessary. The for loop will do the same thing, this `if` can just be removed.

Reply:
I did this so that if the config is a single line, it doesn't say "error on line 1: ", because obviously that is the case. I suppose the likelihood of a single line config isn't that great, and either way it wouldn't really matter. Since this is more of a nit, I think I'll opt to keep it as is, unless you have any other objection.

Reply:
Sure, that's fine.